lib/octokit/client/contents.rb

```diff
@@ -156,7 +156,7 @@ def delete_contents(repo, path, message, sha, options = {})
       # @example Get archive link for octokit/octokit.rb
       #   Octokit.archive_link("octokit/octokit.rb")
       def archive_link(repo, options={})
-        repo_ref = options.delete :ref
+        repo_ref = CGI.escape(options.delete :ref)
```

Comment:
We recently merged in a change to remove many of the references to CGI in favor of ERB.  ERB actually URL encodes things closer to the defined spec like spaces to %20 instead of `+` like CGI does.  What are your thoughts / feels on using ERB instead?

```suggestion
        repo_ref = ERB::Util.url_encode(options.delete :ref)
```

Reply:
Makes sense to me!